### ISSUE TITLE
Build Your Welfare Screen

### ISSUE THREAD

**Issue by lukesleeman** [Western-Health-Covid19-Collaboration/wh_covid19_app#30]
Under "Look after yourself" the following subscreen needs building:

<img width="289" alt="image" src="https://user-images.githubusercontent.com/902252/77297264-20dcdf00-6d3d-11ea-9bf4-94cf5335cced.png">

for the "Phases of Pandemic and suggestions for self-care" button, please link it out to a subscreen which can contain the text "TODO"

**Comment by lukesleeman** [comment]
👋 @greggmiller do you know what phone number the phone icon on this screen should go to?

**Comment by adamkoch** [comment]
Initial build done. Still need correct phone number and to put in the icons.

**Comment by lukesleeman** [comment]
@adamkoch Is the phone number you need for the little Phone icon?

I would say don't worry about that now - the phone buttons on each screen will a feature that comes along later.  I'll fire up a new issue for doing that a little later this evening.

Wondering if we could close out this issue now?

**Comment by adamkoch** [comment]
Ah got it, actually @alessandrofavero updated the phone icon to go to the contacts web link already in https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/pull/58 so we're all set there.

Can close this one out once this is merged:
https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/pull/61
